integration

```diff
@@ -315,6 +315,7 @@
   "erikkastelec/hass-WEM-Portal",
   "eseglem/hass-wattbox",
   "evantaur/seiverkot-consumption",
+  "evercape/hass-resol-KM2",
```

Comment:
hassfest action is missing in the repository and failing against the latest release.

Reply:
Thanks - I was on vacation and did not update anything, but the HASS action has been activated again. Somehow it was deactivated by GITHUB.